### PR TITLE
Match characters  intended to be combined with another character to s…

### PR DESCRIPTION
**Pull request by norkans7**
Addresses https://github.com/rapidpro/rapidpro/issues/898

### CODE REVIEW

File: utils/text_test.go [hunk 1]
```diff
@@ -42,7 +42,10 @@ func TestTokenizeString(t *testing.T) {
 		{"math+=×÷√∊", []string{"math", "+", "=", "×", "÷", "√", "∊"}},     // math symbols treated as individual tokens
 		{"emoji😄🏥👪👰😟🧟", []string{"emoji", "😄", "🏥", "👪", "👰", "😟", "🧟"}},   // emojis treated as individual tokens
 		{"👍🏿 👨🏼", []string{"👍", "🏿", "👨", "🏼"}},                            // tone modifiers treated as individual tokens
-		{"ℹ︎ ℹ️", []string{"ℹ", "ℹ"}},                                      // variation selectors ignored
+		{"ℹ ℹ️", []string{"ℹ", "ℹ️"}},                                      // variation selectors ignored
```
Comment (norkans7) [inline utils/text_test.go]: FWIW It seems the first i had some non printable character next to it, I had to remove that as well both here and in expressions

Comment (nicpottier) [reply utils/text_test.go]: What does this tokenize to? `หอบาเบลทั่วแผ่นดินโลกมีภาษาเดียวและมีสำเนียงเดียวกัน`

Comment (norkans7) [reply utils/text_test.go]: that หอบาเบลทั่วแผ่นดินโลกมีภาษาเดียวและมีสำเนียงเดียวกัน

File: utils/text.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 var snakedChars = regexp.MustCompile(`[^\p{L}\d_]+`)
 
 // treats sequences of letters/numbers/_/' as tokens, and symbols as individual tokens
-var wordTokenRegex = regexp.MustCompile(`[\pL\pN_']+|\pS`)
+var wordTokenRegex = regexp.MustCompile(`[\pM\pL\pN_']+|\pS`)
```
Comment (rowanseymour) [inline utils/text.go]: FYI this includes...

https://www.fileformat.info/info/unicode/category/Mn/list.htm
https://www.fileformat.info/info/unicode/category/Mc/list.htm
https://www.fileformat.info/info/unicode/category/Me/list.htm

There's some weird stuff in `Me`.. but maybe for simplicity best to just match all of `M` like you have it above.

File: utils/text_test.go (hunk 1 above)
Comment (nicpottier) [reply utils/text_test.go]: Ya, so we kind of have to accept that we can't really tokenize these languages. Still think this PR is the right thing but we aren't really breaking up words with our tokenize here, rather just NOT breaking up individual characters. (that sentence above translates to "The Babel tower throughout the earth has one language and has the same accent.")

Comment (norkans7) [reply utils/text_test.go]: Yep

**Pull request merged**